docs/development.md

```diff
@@ -58,7 +58,17 @@ pip-compile base_frozen.in
 
 ## Development process
 
-TODO
+At the beginning set up infrastructure for your test project.
+
+Then implement your changes for bigflow and push changes on your newly created specific branch.
+
+After that, on test project set in `requirements.in` bigflow version from your branch. 
+to do that you just need to set bigflow in `requirements.in` file.
```

Comment:
I'm wondering if code in points wouldn't be more clear for users? 
1. in req.in replace bigflow with git+https://github.com/allegro/bigflow@branch_foo_bar#egg=bigflow[bigquery,dataflow]
2. rm ./resources/requirements.txt
3. run `bf build-requirements`
4. pip uninstall bigflow
5. pip install -r resources/requirements.txt

Reply:
Detailed documentation is in app engine docs. In here it's just quick story because for example we cannot show to open public first point

Reply:
discussed on call, it's fine